docs/process/release.md

```diff
@@ -128,6 +128,14 @@ git tag 'vXX'
 git push upstream && git push upstream --tags
 ```
 
+Double check that the packages we are about to publish are versioned correctly
+by running the following command.
+Their versions are derived from the git tag that we just created.
+
+```
+yarn admin release
```

Comment:
Does this just print out some metadata? It sounds like a very scary command to run and I'm not sure I would be comfortable executing this without fully understanding what it does.

Could we rename it, or at least document more clearly that this won't actually trigger a release? Or maybe it does and I'm just not understanding how this works.

Reply:
I've created another PR to cleanup and rename `scripts/release.ts`.
https://github.com/angular/angular-cli/pull/18184